packages/design-system-tailwind-preset/src/typography.test.ts

```diff
@@ -20,6 +20,61 @@ describe('Typography', () => {
     '--typography-l-heading-sm-regular-line-height',
     '--typography-l-heading-sm-regular-font-weight',
     '--typography-l-heading-sm-regular-letter-spacing',
+    '--typography-s-body-lg-regular-font-family',
```

Comment:
I've added all the removed variables to the ignore list. We have a test in place to ensure all typography variables are used. In a future design-tokens version, these variables will be removed as they are now deprecated.